lib/b2b/index.ts

```diff
@@ -80,8 +80,8 @@ export type {
 
 export type {
   B2BOrganizationsMembersOAuthProvidersGoogleResponse,
-  B2BOrganizationsMembersOAuthProvidersMicrosoftRequest,
```

Comment:
I think this is generated since I don't recall ever having to manually edit it - @logan-stytch do you know why it might not be exporting the type here?

Reply:
Oh wait, there's a section for exporting the manual types. I'll add it